resources/views/layouts/admin.blade.php

```diff
@@ -2,213 +2,141 @@
-					<ul class="nav nav-pills nav-sidebar flex-column" data-widget="treeview" role="menu" data-accordion="false">
-						<li class="nav-item has-treeview menu-open">
-							<a href="#" class="nav-link active">
-								<i class="nav-icon fas fa-tachometer-alt"></i>
-								<p>
-									Starter Pages
-									<i class="right fas fa-angle-left"></i>
-								</p>
-							</a>
-							<ul class="nav nav-treeview">
-								<li class="nav-item">
-									<a href="#" class="nav-link active">
-										<i class="far fa-circle nav-icon"></i>
-										<p>Active Page</p>
-									</a>
-								</li>
-								<li class="nav-item">
-									<a href="#" class="nav-link">
-										<i class="far fa-circle nav-icon"></i>
-										<p>Inactive Page</p>
-									</a>
-								</li>
-							</ul>
-						</li>
-						<li class="nav-item">
-							<a href="#" class="nav-link">
-								<i class="nav-icon fas fa-th"></i>
-								<p>
-								    Simple Link
-								    <span class="right badge badge-danger">New</span>
-								</p>
-							</a>
-						</li>
-					</ul>
-				</nav>
-			</div>
-		</aside>
-		<div class="content-wrapper">
-			<div class="content-header">
-				<div class="container-fluid">
-					<div class="row mb-2">
-						<div class="col-sm-6">
-							<h1 class="m-0 text-dark">ABCDEF Admin</h1>
-						</div>
-						<div class="col-sm-6">
-							<ol class="breadcrumb float-sm-right">
-								<li class="breadcrumb-item"><a href="#">Admin</a></li>
-								<li class="breadcrumb-item active">@yield('title')</li>
-							</ol>
-						</div>
-					</div>
-				</div>
-			</div>
-			<div class="content">
-				<div class="container-fluid">
-					@yield('content')
-				</div>
-			</div>
-		</div>
-		<aside class="control-sidebar control-sidebar-dark">
-			<div class="p-3">
-				<h5>Title</h5>
-				<p>Sidebar content</p>
-			</div>
-		</aside>
-		<footer class="main-footer">
-			<div class="float-right d-none d-sm-inline">
-				Anything you want
-			</div>
-			<strong>Copyright &copy; 2014-2019 <a href="https://adminlte.io">AdminLTE.io</a>.</strong> All rights reserved.
-		</footer>
-	</div>
-	<script src="{{ asset('bower_components/adminlte3/plugins/jquery/jquery.min.js') }}"></script>
-	<script src="{{ asset('bower_components/adminlte3/plugins/bootstrap/js/bootstrap.bundle.min.js') }}"></script>
-	<script src="{{ asset('bower_components/adminlte3/dist/js/adminlte.min.js') }}"></script>
-	@yield('js')
+            </form>
+            <ul class="navbar-nav ml-auto">
+            	<li class="nav-item dropdown">
+                    <a class="nav-link" data-toggle="dropdown" href="#">
+                        <i class="far fa-bell"></i>
+                        <span class="badge badge-warning navbar-badge"></span>
+                    </a>
+                    <div class="dropdown-menu dropdown-menu-lg dropdown-menu-right">
+                        <span class="dropdown-header"> {{ trans('home.noti_num', ['number' => '0']) }} </span>
```

Comment:
số 0 cho vào config

Reply:
the de em bo luon so 0 di tai vi chua dung den